examples/openshift-origin/create.sh

```diff
@@ -110,12 +110,12 @@ echo "sudo cat /srv/kubernetes/server.key; exit;" | gcloud compute ssh ${KUBE_GC
 # The following insertion will fail if indentation changes
 sed -i -e 's/publicKeyFiles:.*$/publicKeyFiles:/g' -e '/publicKeyFiles:/a \ \ - serviceaccounts.private.key' ${OPENSHIFT_CONFIG}/master-config.yaml
 
-docker run -it --privileged -e="KUBECONFIG=/config/admin.kubeconfig" -v ${OPENSHIFT_CONFIG}:/config openshift/origin cli secrets new openshift-config /config -o json &> ${OPENSHIFT_EXAMPLE}/secret.json
-kubectl create -f ${OPENSHIFT_EXAMPLE}/secret.json --namespace='openshift-origin'
+docker run -it --privileged -e KUBECONFIG=/kubeconfig -v ${HOME}/.kube/config:/kubeconfig -v ${OPENSHIFT_CONFIG}:/config openshift/origin:v1.0.3 cli secrets new openshift-config /config -o json &> ${OPENSHIFT_EXAMPLE}/secret.json
```

Comment:
I'm copying my comments from [this comment](https://github.com/kubernetes/kubernetes/issues/16901#issuecomment-154559221) on the issue:

 I noticed three differences between your script and the one currently in the README:
* change `-e=KUBECONFIG=/config/admin.kubeconfig` to `-e KUBECONFIG=/kubeconfig` and add `-v ${HOME}/.kube/config:/kubeconfig`
* change `openshift/origin` to `openshift/origin:v1.0.3`
* change `examples/openshift-origin/secret.json` to `${OPENSHIFT_EXAMPLE}/secret.json`

I'm unclear as why the first change is necessary - OpenShift should generate the necessary `admin.kubeconfig` in the `${OPENSHIFT_CONFIG}` directory.
The second change I'm worried about - we will need to update this every time a different version is built. We should instead find a way to reliably name the binary.
The third change will not be accepted in a PR here as there is an automated bot that make sure that example files written about in READMEs exist in a path, if I remember correctly.